docs/content/observatorium/thanos/thanos_programmatic_access.md

```diff
@@ -0,0 +1,24 @@
+# Thanos Metrics Programmatic Access
+
+The Thanos metrics can be queried using the [this](https://prometheus-api-client-python.readthedocs.io/) Python client library.
+You can install the latest version with `pip install prometheus-api-client`.
+You will need to extract the OpenShift bearer token for the client library to authenticate the [thanos](https://thanos-query-frontend-opf-observatorium.apps.smaug.na.operate-first.cloud/) instance.
+
+### Personal token
+
+You can use your personal token for testing and ad-hoc scripts, but it will expire in ~24 hours.
+
+1. Login to https://console-openshift-console.apps.smaug.na.operate-first.cloud/ with your credentials
```

Comment:
@4n4nd , Is it possible to add a statement for **Log in with** `kube::admin` or `operate-first`?

Reply:
updated :+1: